x-pack/plugins/apm/server/routes/service_groups/route.ts

```diff
@@ -39,6 +38,67 @@ const serviceGroupsRoute = createApmServerRoute({
   },
 });
 
+const serviceGroupsWithServiceCountRoute = createApmServerRoute({
+  endpoint: 'GET /internal/apm/service_groups/services_count',
+  params: t.type({
+    query: rangeRt,
+  }),
+  options: {
+    tags: ['access:apm'],
+  },
+  handler: async (
+    resources
+  ): Promise<{ servicesCounts: Record<string, number> }> => {
+    const { context, params } = resources;
+    const {
+      savedObjects: { client: savedObjectsClient },
+      uiSettings: { client: uiSettingsClient },
+    } = await context.core;
+
+    const {
+      query: { start, end },
+    } = params;
+
+    const [setup, maxNumberOfServices] = await Promise.all([
+      setupRequest(resources),
+      uiSettingsClient.get<number>(apmServiceGroupMaxNumberOfServices),
+    ]);
+
+    const serviceGroups = await getServiceGroups({
+      savedObjectsClient,
+    });
+
+    const serviceGroupsWithServiceCount = await Promise.all(
+      serviceGroups.map(
+        async ({
+          id,
+          kuery,
+        }): Promise<{ id: string; servicesCount: number }> => {
```

Comment:
Is this type annotation necessary ?

Reply:
It seems that it isn't, we can remove it